aten/src/ATen/native/DispatchStub.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <c10/core/DeviceType.h>
-#include <c10/macros/Export.h>
+#include <c10/macros/Macros.h>
```

Comment:
Are these changes auto-generated, or did you need to do a lot of manual tweaking and reverting of changes made by iwyu?

Reply:
All changes are checked and applied by hand. iwyu generates lots of transitive inclusions but we care more about removing unused ones.